easy_ll/src/lib.rs

```diff
@@ -133,27 +139,27 @@ pub fn compile_module(code: &str, bc_file: Option<&[u8]>) -> Result<CompiledModu
         let mut result = CompiledModule {
             context: context,
             engine: None,
-            function: None,
+            run_function: None,
         };
 
         // Parse the IR to get an LLVMModuleRef
-        let module = try!(parse_module_str(context, code));
+        let module = parse_module_str(context, code)?;
 
         if let Some(s) = bc_file {
-            let bc_module = try!(parse_module_bytes(context, s));
+            let bc_module = parse_module_bytes(context, s)?;
             llvm::linker::LLVMLinkModules2(module, bc_module);
         }
 
         // Validate and optimize the module
-        try!(verify_module(module));
-        try!(check_run_function(module));
-        try!(optimize_module(module));
+        verify_module(module)?;
+        check_run_function(module)?;
+        optimize_module(module)?;
 
         // Create an execution engine for the module and find its run function
-        let engine = try!(create_exec_engine(module));
+        let engine = create_exec_engine(module)?;
 
         result.engine = Some(engine);
-        result.function = Some(try!(find_run_function(engine)));
+        result.run_function = Some(find_function(engine, "run")?);
```

Comment:
Can you add a note somewhere that the entry point is still a function called `run`?